provisionersdk/proto/provisioner.proto

```diff
@@ -266,6 +268,7 @@ message PlanComplete {
     repeated Resource resources = 2;
     repeated RichParameter parameters = 3;
     repeated ExternalAuthProviderResource external_auth_providers = 4;
+    repeated Timing timings = 6;
```

Comment:
If I'm not mistaken, adding new protobuf fields here will mean that folks will need to bump external provisioners at the same time as coder?

Reply:
AFAIK all proto fields are nilable for BC, and additional fields should be ignored; but I will validate that.
Good call 👍

Reply:
Validated 👍

Reply:
For future reference, here's how I validated:

- added a log line in the provisioner code to show that timing data was captured
- `make build-fat` from this branch, to produce bins capable of running `provisionerd start --psk=hellocoderd`
- switched to `main`
- ran `develop.sh` with `CODER_PROVISIONER_DAEMON_PSK=hellocoderd` and `CODER_PROVISIONER_DAEMONS=0` to force all builds to the external provisioner
- added our internal enterprise license to allow external provisioners
- created a workspace
- all worked out fine
- switched back to this branch
- ran `develop.sh` with the same args
- created another workspace
- timings were set as expected